tutorials/formatter/README.md

```diff
@@ -0,0 +1,59 @@
+##CsvFormatter Example
+
+This example shows how to use CsvFormatter to customize the delimiters and the schema string for formatting.  
+
+`CsvFormatter` converts the incoming POJO to CSV(by default) or custom('|', ':' etc.) separated string.  
+
+Accordingly, we have additional config file for formatting preferences (`/src/main/resources/schema.json`) besides the common properties file (`/src/main/resources/META-INF/properties.xml`). 
+
+Users can choose the application and additional configuration file to use during launch time. In this example, we use the files mentioned above to customise the schema and configure the operator properties.
+
+
+####**Update Properties from properties.xml - This is needed to run the example:**
+
+- Update these common properties in the file `/src/main/resources/META-INF/properties.xml`:
+
+| Property Name  | Description |
+| -------------  | ----------- |
+| dt.application.CustomOutputFormatter.operator.jsonParser.prop.sleepTime |sleep time for the container |
+| dt.application.CustomOutputFormatter.operator.jsonParser.port.out.attr.TUPLE_CLASS | expected POJO object output of the JSONParser operator |
+| dt.application.CustomOutputFormatter.operator.HDFSOutputOperator.prop.filePath | output file path for the records after formatting |
+| dt.application.CustomOutputFormatter.operator.HDFSOutputOperator.prop.outFileName | output file name for the records to be written after formatting |
+
+
+####**Update Properties from Application.java - This is needed to customise Formatter:**
+
+```
+    CsvFormatter formatter = dag.addOperator("formatter", CsvFormatter.class);
+    formatter.setSchema(SchemaUtils.jarResourceFileToString(filename));
```

Comment:
Not clear what is filename. We need to specify that it has a schema and is part of the java project